### DISCUSSION TITLE
Strelka doesn't generate alerts

### DISCUSSION THREAD

**Discussion by AhmedMontaserT** [Security-Onion-Solutions/securityonion#6695] [Unsupported Versions] [answered]
strelka can analysis files and it saves them in /nsm/strelka/processed 
but there is no alerts in analyst mode

**Comment by dougburks** [comment, marked as answer]
If you go to the Hunt interface and choose the Strelka query from the drop-down box, do you see Strelka logs?
https://docs.securityonion.net/en/2.3/strelka.html#logs

**Reply by AhmedMontaserT** [reply]
yes i can see the logs but there is no alerts

**Reply by dougburks** [reply]
If you see Strelka logs, then Strelka is working correctly. Strelka only generates alerts if there are YARA matches:
https://docs.securityonion.net/en/2.3/strelka.html#alerts
